scripts/init_db.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
-if [ ! -d "migrations" ]; then
-  echo >&2 "❌ Error: '/migrations' folder not found."
-  echo >&2 "Please run this script from the 'pg_replicate/api' directory."
+if [ ! -d "api/migrations" ]; then
```

Comment:
Moved the script in the root folder since we can use it also to setup the database for testing, so it's across crates.